Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -981,6 +978,7 @@ void WebAutomationSession::wheelEventsFlushedForPage(const WebPageProxy& page)
 void WebAutomationSession::didCreatePage(WebPageProxy& page)
 {
     m_bidiProcessor->browserAgent().didCreatePage(page);
+    emitContextCreatedEvent(page);
```

Comment:
I'm not understanding how this recursively emitting context created is supposed to work. This is the only place it gets called, and at this point we'd never have any subframes yet. should `contextCreatedForFrame` be calling into the recursively emit events too?